deepchem/models/tests/test_jax_model.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+
+from deepchem.models.tests.test_graph_models import get_dataset
+import numpy as np
+
+try:
+  import jax
+  import jax.numpy as jnp
+  import haiku as hk
+  import optax
+  from deepchem.models import JaxModel
+  has_haiku_and_optax = True
+except:
+  has_haiku_and_optax = False
+
+
+@unittest.skipIf(not has_haiku_and_optax,
+                 'Jax, Haiku, or Optax are not installed')
+def test_jax_model_for_regression():
```

Comment:
Should we add a `TODO` comment here on additional methods that will be tested after they're written?